frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/pipeline-as-code.ts

```diff
@@ -13,23 +13,27 @@ import {
 } from '@console/dev-console/integration-tests/support/pages';
 import { pipelineActions, pipelineTabs, repositoryDetailsTabs } from '../../constants';
 import { createRepositoryPO } from '../../page-objects';
-import { pipelinesPage, repositoryDetailsPage, repositoriesPage } from '../../pages';
+import { repositoryDetailsPO } from '../../page-objects/pipelines-po';
+import {
+  pipelinesPage,
+  repositoryDetailsPage,
+  repositoriesPage,
+  pipelineRunDetailsPage,
+} from '../../pages';
 import { actionsDropdownMenu, tableFunctions } from '../../pages/functions/common';
 
 Given('user has installed pipelines as code', () => {
   // Steps to install pipeline as code feature : https://gist.github.com/chmouel/272df2cfbeef5606ca36d29a9a2d2f96
-  cy.exec(
-    `oc apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-0.1/release-0.1.yaml`,
-    {
-      failOnNonZeroExit: false,
-    },
-  );
+  cy.exec(`oc apply -f ${'testData/repository-crd-testdata/pac-installation-release0.1.yaml'}`, {
+    failOnNonZeroExit: false,
+  });
```

Comment:
With Redhat Openshift Pipeline 1.7.0 (to be released tomorrow -12th April), all the PAC resources will be packaged and installed with it. I would suggest add this step as an conditional step, so if the Repository tab is already present do not install it again.